.github/workflows/test.yml

```diff
@@ -27,14 +27,27 @@ jobs:
         (New-Object System.Net.WebClient).DownloadFile("http://prdownloads.sourceforge.net/swig/swigwin-4.0.1.zip","swigwin-4.0.1.zip");
         Expand-Archive .\swigwin-4.0.1.zip .;
         echo "$((Get-Item .).FullName)/swigwin-4.0.1" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-    - name: Install APT dependencies
-      if: "startsWith(runner.os, 'ubuntu')"
+    - name: Setup Display
+      if: "startsWith(runner.os, 'Linux')"
       run: |
-        sudo apt-get install -y git build-essential apt-utils wget libfreetype6 libpng-dev libopenblas-dev gcc gfortran libsnappy-dev
+        sudo apt -y install xvfb libgtk-3-dev freeglut3-dev
```

Comment:
`xvfb` is a dependency to create a virtual display for psychopy to not error out when initialized
`libgtk-3-dev` is a requirement for `wxPython`
`freeglut3-dev` is a requirement for a dependency in `wxPython`

Reply:
It'd be nice if you left these useful hints as comments in the file instead (better longevity that way).

Reply:
```suggestion
        # xvfb is a dependency to create a virtual display
        # libgtk-3-dev is a requirement for wxPython
        # freeglut3-dev is a requirement for a wxPython dependency
        sudo apt -y install xvfb libgtk-3-dev freeglut3-dev
```